### ISSUE TITLE
suppress violations

### ISSUE THREAD

**Issue by Khabib73** [wemake-services/dotenv-linter#992]
> We can also add `# dotenv:ignore[ERROR]` comments on top of the values (the line above) that can be false positives :)
> This can be a separate feature. 

 _Originally posted by @sobolevn in [#978](https://github.com/wemake-services/dotenv-linter/issues/978#issuecomment-4831452828)_

It remains only to decide design of this feature:

each ignore is on a new line:

```env
# dotenv:ignore[SpacedNameViolation]
# dotenv:ignore[IncorrectNameViolation]
KEY=VALUE
SOME_KEY=doesn't contain ignore
```

or using a list of violations:

```env
# dotenv:ignore[SpacedNameViolation, IncorrectNameViolation]
KEY=VALUE
SOME_KEY=doesn't contain ignore
```

Please, assign me

**Comment by sobolevn** [comment]
Yes, we 100% needs this :)

**Comment by sobolevn** [comment]
I would go for list option as the only one :)

**Comment by Khabib73** [comment]
OK, I'll do it tomorrow :)

**Comment by Khabib73** [comment]
I also think we should allow using ignore on a new line or in a list, because if there are too many violations in a single list, the line becomes very long and looks messy:

```env
# dotenv:ignore[SpacedNameViolation, IncorrectNameViolation, AnotherViolation, ...]
KEY=VALUE
SOME_KEY=doesn't contain ignore
```

best way:
```env
# dotenv:ignore[AnotherViolation]
# dotenv:ignore[SpacedNameViolation, IncorrectNameViolation]
KEY=VALUE
SOME_KEY=doesn't contain ignore
```

And maybe it's a good style to group ignored violations?

**Comment by sobolevn** [comment]
Ok, sounds reasanoble. 

Let's also add `# dotenv:disable` and `# dotenv:enable` in the next PRs, to disable it not per line, but per file / group of lines.

**Comment by Khabib73** [comment]
like this:

```
# dotenv:disable
KEY1=value1   # this and the following lines are not checked
KEY2=value2
# dotenv:enable
KEY3=value3   # is active again
```
